airflow-core/docs/authoring-and-scheduling/event-scheduling.rst

```diff
@@ -43,7 +43,7 @@ SQS:
 .. code-block:: python
 
     from airflow.sdk import Asset, AssetWatcher
-    from airflow.providers.common.msgq.triggers.msg_queue import MessageQueueTrigger
+    from airflow.providers.common.messaging.triggers.msg_queue import MessageQueueTrigger
```

Comment:
I wonder if this paragraph should be in the provider docs rather than Airflow core docs

Reply:
That is an option yeah, there is a shortened version of this code block in the [provider docs](http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers-common-messaging/stable/triggers.html) already. @vikramkoka @vincbeck that feels like a you decision :)

Reply:
By doing so you are binding the doc update into core releases even though the code sits in providers.
That normally invites troubles.
I didn't suggest to remove it, I suggested to shift it to the provider.
The core docs can refer to further read in providers.. we do that for k8s, celery and others.

Reply:
That's a fair call-out. Indeed, I think referencing the example in core docs would be a good idea. I agree that binding the provider with core releases can lead to some issues.

Reply:
@TJaniF can you make the changes?

Reply:
@eladkal Do you mean merging this PR or creating a new PR removing the example from core docs and linking to provider docs instead?

Reply:
extracting what we need to provider docs and keep the core docs minimal with links to provider